partiql-ast/src/main/java/org/partiql/ast/expr/ExprRowValue.java

```diff
@@ -10,16 +10,45 @@
 import java.util.List;
 
 /**
- * TODO docs, equals, hashcode
- *  Also add optional [ROW] keyword property. https://ronsavage.github.io/SQL/sql-99.bnf.html#row%20value%20constructor
+ * This represents SQL:1999's row value constructor. EBNF:
+ * <code>
+ *     &lt;row value constructor&gt; ::=
+ *      &lt;row value constructor element&gt;
+ *      | [ ROW ] &lt;left paren&gt; &lt;row value constructor element list&gt; &lt;right paren&gt;
+ *      | &lt;row subquery&gt;
+ * </code>
+ * This specifically models the second variant where the keyword {@code ROW} is used.
  */
 @lombok.Builder(builderClassName = "Builder")
 @EqualsAndHashCode(callSuper = false)
 public class ExprRowValue extends Expr {
+    // TODO: Equals and hashcode
+
+    /**
+     * Specifies whether the ROW keyword explicitly precedes the elements in the textual representation. For example,
+     * {@code ROW (1, 2, 3)} versus {@code (1, 2, 3)}. In the first example, {@code isExplicit} is true.
```

Comment:
Could we also add a parse test for this addition? Don't think any of the added DML tests included the explicit `ROW` keyword constructor.